README.md

```diff
@@ -1,6 +1,7 @@
 # csdt2324ki49svyrydiukdv23
```

Comment:
add all task details.

Reply:
what is Radar? we didn't agree with you to change task.

Reply:
My file is described, but the commit does not show the change. Do you see the file description? Should I remake it?

Reply:
I do. But, what is Radar? We didn't agree with you to change task.

Reply:
I made a project using an ARDUINO UNO as a board, and using its elements I made a device for determining the distance to an object. I'm sorry that I didn't agree with you about changing the topic. Can I continue working with this project?